### ISSUE TITLE
Why does the line's loss function use KL-loss alone？

### ISSUE THREAD

**Issue by pangedeshijie** [alibaba/graph-learn#63]
Sigmoid cross entropy loss for unsuperviesd model, And kl-loss used for line. 
But according to the formula **cross_entropy-entropy=KL**,  **entropy** is fixed for the entire training set, so using **cross_entropy** is the same as using **KL**. 
(Of course, for each specific batch, b changes.)

**I don’t quite understand why line uses KL-loss, can you help?**

@archwalker @baoleai 
Thanks

**Comment by archwalker** [comment]
Hi, KL used in LINE is to make the formula consistent with the paper.
